cli.rb

```diff
@@ -4,7 +4,7 @@
 
 def foodwarning(amount)
     uri = URI('https://megov.bayern.de/verbraucherschutz/baystmuv-verbraucherinfo/rest/api/warnings/merged')
-    hash = {}
+    hash =  {}
```

Comment:
Indentation too much
```suggestion
    hash = {}
```